js/src/dapps/static/console.js

```diff
@@ -159,7 +184,7 @@ function exec() {
       let res;
       try {
         res = evaluate(c);
-        if (typeof(res) == 'object' && res !== null && res.constructor.name == "Promise") {
+        if (typeof(res) == 'object' && res !== null && res.constructor && res.constructor.name == "Promise") {
```

Comment:
`===`

Be aware that `res.constructor.name` may not work in FF with the es6-promise polyfills. (Have not checked myself with this branch, but Promise checking this way is always a bit of an issue)